src/modules/course-list/components/skills-list/SkillsList.component.tsx

```diff
@@ -1,14 +1,14 @@
 import { FC } from 'react';
 
-import { ICourseMeta } from '../../@types/types';
+import { ICourseMeta } from '../../../../shared/@types/types';
 
 import './SkillsList.scss';
 
-interface SkillsListProps {
+interface Props {
```

Comment:
I see that you have renamed your interfaces to `Props`. Can you explain it, or maybe provide link to conventions about interface naming

Reply:
https://github.com/Genesis-Front-End-School/clean-code-principes-v4voloshyn/pull/4#discussion_r1184137641
Thats my convention of intarface naming :)